Zadanie 12.3 Wyszukiwarka krajów/js/scripts.js

```diff
@@ -0,0 +1,22 @@
+
+var url = 'https://restcountries.eu/rest/v1/name/';
+var countriesList = $('#countries');
+
+$('#search').click(searchCountries);
+
+function searchCountries() {
+    var countryName = $('#country-name').val();
+if(!countryName.length) countryName = 'Poland';
+$.ajax({
+         url: url + countryName,
+         method: 'GET',
+         success: showCountriesList
+     });
+}
+function showCountriesList(resp) {
+    countriesList.empty();
+    resp.forEach(function(item) {
+        $('<li>').text(item.name).appendTo(countriesList);
+    });
+}
+showCountriesList();
```

Comment:
tutaj tak nie mozesz wywolac tej funkcji - mozesz sprawdzic, ze wyrzuca blad w konsoli przegladarki. Ta funkcja przyjmuje jako argument odpowiedz z serwera, a tutaj nie wie czym jest ta odpowiedz i dlatego nie wie wewnatrz czym jest `resp`. 
Te funkcje wywolujesz juz po sukcesie zapytania wyslanego ajaxem (linia 13) i to wystarczy. Tak wiec tutaj to wywolanie jest niepotrzebne